app/Entirety/projects/views.py

```diff
@@ -18,22 +18,36 @@ class Index(LoginRequiredMixin, ListView):
     template_name = "projects/index.html"
 
     def get_queryset(self):
-        return Project.objects.order_by("date_modified").filter(
-            name__icontains=self.request.GET.get("search", default="")
-        )
+        if self.request.user.is_server_admin or self.request.user.is_project_admin:
```

Comment:
Should Project admin can see all the projects? I think it would be better, if project admins can only see the projects they have access to.